doc/man3/SSL_get_value_uint.pod

```diff
@@ -162,18 +178,82 @@ of QUIC stream creation flow control.
 Can be queried using the convenience macro
 SSL_get_quic_stream_bidi_remote_avail().
 
-=item B<SSL_VALUE_QUIC_STREAM_UNI_REMOTE_AVAIL>
+=item B<SSL_VALUE_QUIC_STREAM_UNI_REMOTE_AVAIL> (connection object)
 
 As above, but provides the number of unidirectional, remotely-initiated streams
 available to be created (but not yet created).
 
 Can be queried using the convenience macro
 SSL_get_quic_stream_uni_remote_avail().
 
+=item B<SSL_VALUE_EVENT_HANDLING_MODE> (connection or stream object)
```

Comment:
Should this be called `SSL_VALUE_QUIC_EVENT_HANDLING_MODE`? It seems to be QUIC specific.

Reply:
This was considered and avoided. This is quite a generic concept and I wouldn't be surprised if we support it for other things in future.